<launch>
<arg name="robot_id" value="$(optenv ROBOT_ID unnamed_robot)"/>
<!-- Follow_Path -->
  <group ns="$(arg robot_id)">
    <node pkg="pileec2223_control_methods" type="GoToXY"
        name="GoToXY" clear_params="true"
        output="$(optenv ROBOT_OUTPUT screen)">
      <rosparam command="load" subst_value="True"
          file="$(find pileec2223_control_methods)/config/GoToXY.yaml"/>
    </node>
  </group>
</launch>
